b2sdk/file_version.py

```diff
@@ -576,7 +592,7 @@ def from_response_headers(self, headers):
             content_length=content_length,
             content_language=headers.get('Content-Language'),
             expires=headers.get('Expires'),
-            cache_control=headers.get('Cache-Control'),
+            cache_control= cache_control,
```

Comment:
```suggestion
            cache_control=cache_control,
```
Extra space violates pep8. This shows that you haven't ran `nox -s format` before submitting the PR and you should really run the entire `nox -s test-3.10` (or whichever python version you are using) to make sure code is formatted properly, unit tests pass and integration tests pass.

Then CI will run it again on many versions of python.